.github/workflows/bump-version.yml

```diff
@@ -39,37 +39,18 @@ jobs:
       - name: Checkout out
         uses: actions/checkout@v3
         with:
-          ref: ${{ inputs.branch || github.ref }}
-      - name: Setup python
-        uses: actions/setup-python@v4
+          ref: ${{ inputs.branch }}
+          fetch-depth: 0
+          token: "${{ secrets.GITHUB_API_TOKEN }}"
+      - id: cz
+        name: Create bump and changelog
+        uses: commitizen-tools/commitizen-action@0.16.3
         with:
-          python-version: "3.9"
-      - name: Setup git
+          github_token: ${{ secrets.GITHUB_API_TOKEN }}
+          branch: "bump-version-test"
```

Comment:
is this for testing only?

Reply:
yeah... eventually we need a scheme for the name... currently my best idea is to use the dev version since we have access to it